cloudify_tf/terraform/__init__.py

```diff
@@ -152,7 +170,14 @@ def state_pull(self):
             return json.loads(pulled_state)
 
     def refresh(self):
-        command = self._tf_command(['refresh', '-no-color'])
+        if self.terraform_version >= "0.15.4":
```

Comment:
just to cover better checking against versions
```suggestion
        from distutils.version import LooseVersion as parse_version
        if parse_version(self.terraform_version) >= parse_version("0.15.4"):
```